addons/event_hr_skills/tests/test_event_hr_skills.py

```diff
@@ -0,0 +1,39 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import Command
+from odoo.tests import common
+
+
+class TestEventHrSkills(common.TransactionCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.alice = cls.env['hr.employee'].create({
+            'name': 'Alice',
+            'email': 'alice@example.com',
+        })
+        cls.training_category_tag = cls.env['event.tag.category'].create({
+            'name': 'Training',
+            'resume_line_type_id': cls.env.ref('hr_skills.resume_type_experience').id,
+            'tag_ids': [
+                Command.create({'name': 'Sponsored MOOC'}),
+                Command.create({'name': 'Internal Training'}),
+            ],
+        })
+        cls.training_tag = cls.training_category_tag.tag_ids[1]
+        cls.frobination_training = cls.env['event.event'].create({
+            'name': 'Frobination Training',
+            'tag_ids': [Command.link(cls.training_tag.id)]
+        })
+
+    def test_regitering_creates_resume_line(self):
+        registration = self.env['event.registration'].create({
```

Comment:
Make a mixed data test without partner, with partner with employee, with partner without employee, to check your batch support :)